livre/core/data/VolumeInformation.h

```diff
@@ -87,15 +87,25 @@ struct VolumeInformation
     /** The normalized size of the volume. */
     Vector3f worldSize;
 
-    /** Real-world size of the volume in meters. */
-    Boxf boundingBox;
+    /** The transformation matrix between data space and livre space */
+    Matrix4f dataToLivreTransform;
+
+    /** The resolution in um/voxel. Only valid for some datasources,
```

Comment:
resolution in data units/voxel?
With meterToDataUnitRatio the actual unit can be inferred.